contracts/testing/TestBucketLender.sol

```diff
@@ -34,7 +34,8 @@ contract TestBucketLender is BucketLender {
         address heldToken,
         address owedToken,
         uint32[7] parameters,
-        address[] trustedMarginCallers
+        address[] trustedMarginCallers,
+        address[] trustedWithdrawers
```

Comment:
You need to bump the version on `package.json` and `package-lock.json` in order to publish these changes. When you do that, it will automatically publish the new stuff to npm and docker hub